scripts/inference_control.py

```diff
@@ -80,7 +81,7 @@ def initUI(self):
         # self.numgpu_lineEdit.setText("1")
         # self.epoch_lineEdit.setText("10")
         # self.tidims_lineEdit.setText("1,3,224,224")
-        # self.dpath_lineEdit.setText("/home/hansel/MIVisionX-Neural-Net-Workflow/sample/imagenet_10dir")
+        # self.dpath_lineEdit.setText("/home/hansel/imagenet_20dir")
```

Comment:
@hansely can you keep this empty? We do not want to check in our local file directory address